com/williamfiset/datastructures/splaytree/SplayTree.java

```diff
@@ -0,0 +1,364 @@
+package com.williamfiset.datastructures.splaytree;
```

Comment:
Can you remove this now that its in the other folder?

Reply:
Done